### PR TITLE
Page "Manage assistants" for Builders should not be in Build menu if Vaults Activated.

### PR DESCRIPTION
**Pull request by PopDaph**
## Description

The page "Manage Assistants" is the only page from the Build menu that we will still use with Vault. 
It is accessible from the chat homepage (button "Manage). 
When we click on it, we don't want to display the Build menu that is supposed to disappear when Vault is activated. 

This PR is a hack that copy this page (reusing the page component) on /assistant/manage-assistants, keeping the Chat menu opened. 

## Risk

Can be rolled back. 

## Deploy Plan

Deploy front.

### CODE REVIEW

File: front/components/assistant/AssistantBrowser.tsx [hunk 1]
```diff
@@ -106,6 +106,11 @@ export function AssistantBrowser({
           ? visibleTabs[0].id
           : null;
 
+  const isVaultsEnabled = owner.flags.includes("data_vaults_feature");
+  const builderManageAssistantsRoute = isVaultsEnabled
+    ? `/w/${owner.sId}/assistant/manage-assistants/`
```
Comment (flvndvd) [inline front/components/assistant/AssistantBrowser.tsx:111]: I'd say it should be `/w/${owner.sId}/assistants`

File: front/pages/w/[wId]/builder/assistants/index.tsx [hunk 2]
```diff
@@ -216,15 +221,30 @@ export default function WorkspaceAssistants({
     }
   }, [tabScope]);
 
+  // Hack to display this page either from Builder or Chat Menu
+  const { conversations, isConversationsError } = useConversations({
```
Comment (flvndvd) [inline front/pages/w/[wId]/builder/assistants/index.tsx:225]: Please add a comment so it uses `ConversationLayout` once the build menu is gone 🙏

**Review by flvndvd — APPROVED** [review]

File: front/components/assistant/AssistantBrowser.tsx (hunk 1 above)
Comment (PopDaph) [reply front/components/assistant/AssistantBrowser.tsx]: Agreed it's better, I added a commit!

File: front/pages/w/[wId]/builder/assistants/index.tsx (hunk 2 above)
Comment (PopDaph) [reply front/pages/w/[wId]/builder/assistants/index.tsx:226]: Done!

**Pull request merged**